doc/source/whatsnew/v0.25.0.rst

```diff
@@ -181,6 +181,15 @@ The repr now looks like this:
     }]
     json_normalize(data, max_level=1)
 
+Column order is preserved when passing a list of dicts to DataFrame
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A long requested feature, this has been the behavior for lists of
+`OrderedDict` since v0.19.0. With dict key-order now officially
```

Comment:
Please keep it a bit "dryer", the "A long requested feature" is not needed (we have many other things which are much more requested)

Reply:
You have many more important things which users requested, but we've established that this feature has a pretty long request history. Still, point taken.